h/api/search/core.py

```diff
@@ -10,10 +10,13 @@
 log = logging.getLogger(__name__)
 
 
-def search(request_params, user=None, search_normalized_uris=False):
+def search(request, request_params, user=None, search_normalized_uris=False):
```

Comment:
If we're passing the request through now, is there any reason to have `request_params`? We can just take `request.params`.